module-1.md

```diff
@@ -0,0 +1,73 @@
+# 1: What is mesh?
+
+Total length:  1.5 hour
+Planned:      1.25 hour
+
+| **Section**                                        | **Length** |
+|----------------------------------------------------|------------|
+| **1: Topic introduction**                          | 15 min     |
+| **2: Hands-on and material exploration**           | 40 min     |
+| **3: Discussion and reflection activity**          | 20 min     |
+| **4: Wrap-up and week ahead**                      | 5 min      |
+
+## 1: Topic introduction
+
+- Introduce yourself, why you are interested in this workshop
+- Discuss familiar topics like net neutrality, internet service providers, and internet censorship
+- Discuss social, political and technical reasons for a mesh
+- Succinct high-level slides on many pieces that will be explored in more detail in upcoming modules:
+    - two minutes on historic Toronto networks
+    - two minutes on community mesh networks in North America and other parts of the world
+    - general landscape of technologies like flashing routers with OpenWrt to install some mesh routing software
+- Set realistic expectations of the state of mesh technology
+
+## 2: Hands-on and material exploration
+
+### Objectives
+
+- Experience local peer-to-peer applications that are relatable to those we are familiar with from the Internet (e.g. Facebook, Dropbox, Whatsapp, Twitter, Spotify)
+- "Run your own app" on pre-configured nodes and contrast "serverless" applications on a distributed physical network vs. traditional centralized edge services via ISP networks
+
+**Ideas for peer-to-peer applications:**
+
+_Preference to those that require minimal steps (e.g. exchanging keys, after `docker start`) and have a familiar UI._
+
+    - Secure Scuttlebutt / Patchwork (need to pin public keys)
+    - IPFS (auto peer but need to share content hash)
```

Comment:
https://hub.docker.com/r/jkms/rpi-go-ipfs/

this is tested and working using HypriotOS on my RPI3. GH here: https://github.com/jkms/go-ipfs